partiql-lang/src/test/kotlin/org/partiql/lang/eval/EvaluatingCompilerGraphMatchTests.kt

```diff
@@ -14,7 +14,292 @@ class EvaluatingCompilerGraphMatchTests : EvaluatorTestBase() {
     // So far, graphs are only supported in COMPILER_PIPELINE
     private val currentPipeline = EvaluatorTestTarget.COMPILER_PIPELINE
 
-    private val session = sessionOf(
+    // A "template" for the subsequent parameterized tests.
+    fun testGraphQueries(
+        session: EvaluationSession,
+        qr: Pair<String, String>
+    ) {
+        val (query, result) = qr
+        runEvaluatorTestCase(
+            EvaluatorTestCase(
+                query = query,
+                expectedResult = result,
+                targetPipeline = currentPipeline
+            ),
+            session
+        )
+    }
+
+    /******** Directionality ********/
+
+    val sessionDirectionality = sessionOf(
+        graphs = mapOf(
+            "pairs" to """{ nodes: [ {id: a1, labels:["a1"], payload: "A1"}, 
+                |                    {id: b1, labels:["b1"], payload: "B1"},
+                |                    {id: a2, labels:["a2"], payload: "A2"},
+                |                    {id: b2, labels:["b2"], payload: "B2"} ], 
+                |           edges: [ {id: d1, labels:["d1"], payload: "D1", ends: (a1 -> b1)},
+                |                    {id: u2, labels:["u2"], payload: "U2", ends: (a2 -- b2)}  ] 
+                |         }""".trimMargin()
+        )
+    )
+
+    @ParameterizedTest
+    @ArgumentsSource(Directionality::class)
+    fun testDirectionality(qr: Pair<String, String>) {
+        testGraphQueries(sessionDirectionality, qr)
+    }
+
+    class Directionality : ArgumentsProviderBase() {
```

Comment:
Will these tests still remain in `partiql-lang-kotlin` after https://github.com/partiql/partiql-tests/pull/91 is merged into `partiql-tests`? Seems like a duplication of tests.

Or perhaps are we waiting for the Kotlin conformance test runner improvements that we spoke about in https://github.com/partiql/partiql-lang-kotlin/pull/1096 before relying just on `partiql-tests`?

Reply:
> For tests that have been previously ported from kotlin to conformance, have originals always been removed?

Tests that were part of `pts` were removed (https://github.com/partiql/partiql-lang-kotlin/pull/910). The other tests haven't yet been removed yet (e.g. porting from `EvaluatorTestSuite.kt`.

---

Agree with the point that the test runner would need improvements in running the tests and modeling passing/failing tests before we go through with deleting the other tests.